stock_location_warehouse/models/stock_location.py

```diff
@@ -0,0 +1,25 @@
+from collections import OrderedDict
+
+from odoo import api, fields, models
+
+
+class StockLocation(models.Model):
+
+    _inherit = "stock.location"
+    warehouse_id = fields.Many2one(
+        comodel_name="stock.warehouse", compute="_compute_warehouse_id"
```

Comment:
@BernatPForgeFlow Nice module!!

Any reason why you didn't make the field stored? Think it would be easy to have the field available in the database and UI thing like grouping?

Reply:
@CasVissers-360ERP Yes, nice observation. I deleted the stored value because the previous function had nested loops with locations. Now we have exactly backported just what odoo15 does and it works fine. If I put `store=True`, then code stops working and it would need to be refactored again, and it doesn't seem trivial to me.

Reply:
@CasVissers-360ERP Finally the code is refactored and the field `warehouse_id` is stored. How do you see it?